reports/GoodsAndServiceTax/BaseViewConfig.js

```diff
@@ -7,76 +9,46 @@ export default {
       label: 'Transfer Type',
       fieldname: 'transferType',
       options: [
-        '',
         'B2B',
         'B2C-Large',
         'B2C-Small',
-        'Nil Rated, Exempted and Non GST supplies'
+        'Nil Rated, Exempted and Non GST supplies',
       ],
-      size: 'small'
+      default: 'B2B',
+      size: 'small',
     },
     {
       fieldtype: 'Data',
       label: 'Place',
       size: 'small',
       placeholder: 'Place',
-      fieldname: 'place'
+      fieldname: 'place',
     },
     {
       fieldtype: 'Date',
       label: 'From Date',
       size: 'small',
       placeholder: 'From Date',
-      fieldname: 'fromDate'
+      fieldname: 'fromDate',
+      default: DateTime.local().minus({ months: 3 }).toISODate(),
```

Comment:
This should be a Function, cause `DateTime` will be evaluated on startup, so if someone leaves the software running (i.e. no restart), default will contain stale values.